src/Microsoft.ML.Data/Transforms/ConversionsCatalog.cs

```diff
@@ -24,7 +24,7 @@ public static class HashingEstimatorCatalog
         /// <param name="outputColumn">Name of the column to be transformed. If this is null '<paramref name="inputColumn"/>' will be used.</param>
         /// <param name="hashBits">Number of bits to hash into. Must be between 1 and 31, inclusive.</param>
         /// <param name="invertHash">Limit the number of keys used to generate the slot name to this many. 0 means no invert hashing, -1 means no limit.</param>
-        public static HashingEstimator Hash(this TransformsCatalog.Conversions catalog, string inputColumn, string outputColumn = null,
+        public static HashingEstimator Hash(this TransformsCatalog.ConversionTransforms catalog, string inputColumn, string outputColumn = null,
```

Comment:
nit - Why rename this? TransformsCatalog.ConversionTransforms sounds a bit redundant as this sounds like there is a TransformsCatalog and that Conversions is related to transforms that deal with converting.

Reply:
If you look at the TransformsCatalog, i standardize the naming there. This is an effect of that.  

---
In reply to: [230153450](https://github.com/dotnet/machinelearning/pull/1494#discussion_r230153450) [](ancestors = 230153450)